test/Tests.Infrastructure/RavenTestBase.Cluster.cs

```diff
@@ -39,6 +45,13 @@ public async Task WaitForRaftCommandToBeAppliedInLocalServerAsync(string command
             await _parent.Server.ServerStore.Cluster.WaitForIndexNotification(updateIndex, TimeSpan.FromSeconds(10));
         }
 
+        public async Task<PutIndexResult> CreateIndexInClusterAsync(IDocumentStore store, AbstractIndexCreationTask index)
+        {
+            var results = await store.Maintenance.ForDatabase(store.Database).SendAsync(new PutIndexesOperation(index.CreateIndexDefinition()));
+            var filterredResults = results.Where(r => r.Index == index.IndexName).ToArray();
+            return filterredResults.Length == 1 ? filterredResults[0] : null;
```

Comment:
move `await WaitForRaftIndexToBeAppliedInClusterAsync(result.RaftCommandIndex);` to here as well

Reply:
Done